docs/sources/administration/grafana-advisor/_index.md

```diff
@@ -96,6 +96,112 @@ If the [Grafana LLM app](https://grafana.com/grafana/plugins/grafana-llm-app/) i
 
 ![<Grafana Advisor - LLM suggestions>](/media/docs/grafana-advisor/llm-suggestions.png)
 
+## How to Create an Alert Based on Advisor Results
+
+This guide will walk you through creating a Grafana alert that monitors Advisor check results and triggers when failures are detected.
+
+## Prerequisites
+
+- Admin access to your Grafana instance
+
+## Step 1: Create a Service Account and Token
+
+1. Navigate to **Administration → Users and access → Service accounts** in your Grafana instance
+2. Click **Add service account**
+3. Provide a name (e.g., "advisor-alert-service-account")
+4. Set the role to **Admin** to ensure proper permissions
+5. Click **Create**
+6. In the service account details, click **Add service account token**
+7. Provide a token name and set an appropriate expiration
+8. Click **Generate token**
+
+> **Important**: Copy the token value immediately and store it securely - you won't be able to see it again
+
+## Step 2: Install Grafana Infinity plugin (if not already installed)
+
+1. Go to **Administration → Plugins**
+2. Search for "Infinity"
+3. If not installed, click **Install**
+
+## Step 3: Create an Infinity data source
+
+1. Navigate to **Connections → Data sources**
+2. Click **Add data source**
+3. Search for and select **Infinity**
+4. Configure the data source:
+   - **Name**: Give it a descriptive name (e.g., "Advisor API")
+   - **Authentication**: In the **Auth details**, select **Bearer Token**. In the **Auth details** section, paste the service account token from Step 1 and in the **Allowed hosts** section, write your Grafana app URL and click the "Add" button (e.g., `https://your-grafana-host.com`).
+5. Click **Save & test** to verify the connection
+
+## Step 4: Create the Alert Rule
+
+Now you have everything you need to create an alert based on Advisor results.
+
+### 4.1 Basic Alert Setup
+
+1. Navigate to **Alerting → Alert rules**
+2. Click **New alert rule**
+3. Provide a rule name (e.g., "Advisor Failures Alert")
+
+### 4.2 Configure the Query
```

Comment:
(not reviewed this section, waiting on version update to plugin). I wonder if providing provisioning example may make this easier?